### PR TITLE
Upgrade to React 0.14.2

### PR DESCRIPTION
**Pull request by kassens**
- update package.json
- React.findDOMNode » ReactDOM.findDOMNode
- React.addons.createFragment » react-addons-create-fragment
- update `.flowconfig` (see facebook/fbjs#44)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,9 @@
     "fbjs-scripts": "0.2.1",
     "immutable": "^3.7.4",
     "object-assign": "^3.0.0",
-    "react": "^0.13.3"
+    "react": "0.14.0",
```
Comment (yungsters) [inline package.json]: Maybe `^0.14.0`?

File: frontend/DataView/Simple.js [hunk 2]
```diff
@@ -75,7 +76,7 @@ class Simple extends React.Component {
   }
 
   selectAll() {
-    var node = React.findDOMNode(this.input);
+    var node = ReactDOM.findDOMNode(this.input);
```
Comment (spicyj) [inline frontend/DataView/Simple.js]: don't need findDOMNode at all here

**Comment by kassens** [comment]
Thanks for the review!
- Removed the rest of the `findDOMNode` calls that weren't needed.
- Used `^0.14.0` as the versions.

**Comment by kassens** [comment]
ping @yungsters, @spicyj

**Comment by spicyj** [comment]
:+1:

**Pull request merged**